src/main/java/pro/sky/telegrambotanimalshelter/controller/HumanCatController.java

```diff
@@ -0,0 +1,114 @@
+package pro.sky.telegrambotanimalshelter.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import pro.sky.telegrambotanimalshelter.models.HumanCat;
+import pro.sky.telegrambotanimalshelter.service.implementation.HumanCatServiceImpl;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("human-cat")
+public class HumanCatController {
+
+    private final HumanCatServiceImpl personCatService;
+
+    // Конструктор контроллера, который принимает сервис в качестве зависимости.
+    public HumanCatController(HumanCatServiceImpl personCatService) {
```

Comment:
```suggestion
    public HumanCatController(HumanCatService personCatService) {
```
зависеть надо от абстракций, а не реализаций